packages/lesswrong/components/ea-forum/TargetedJobAd.tsx

```diff
@@ -221,247 +184,111 @@ type EAGOccupation =
+  coreTagReadsThreshold?: number,               // used to adjust the threshold for how many post reads per topic to qualify for seeing the ad
+  logo: string,                                 // url for org logo
+  occupation: string,                           // text displayed in the tooltip
+  feedbackLinkPrefill: string,                  // url param used to prefill part of the feedback form
+  bitlyLink: string,                            // bitly link to the job ad page
   role: string,
-  insertThe?: boolean,                                        // set if you want to insert a "the" before the org name
+  insertThe?: boolean,                          // set if you want to insert a "the" before the org name
   org: string,
-  orgLink: string,                                            // internal link on the org name
+  orgLink: string,                              // internal link on the org name
   salary?: string,
   location: string,
-  countryCode?: string,                                       // if provided, only show to users who we think are in this country
-  roleType?: string,                                          // i.e. part-time, contract
-  deadline?: moment.Moment,                                   // also used to hide the ad after this date
-  getDescription: (classes: ClassesType) => JSX.Element,
-  get80kDescription?: (classes: ClassesType) => JSX.Element   // just used on some ads for an A/B test
+  countryCode?: string,                         // if provided, only show to users who we think are in this country (match on account location)
+  countryName?: string,                         // if provided, only show to users who we think are in this country (match on EAG data)
+  nearestCity?: string,                         // if provided, only show to users who we think are near this city (match on EAG data)
+  roleType?: string,                            // i.e. part-time, contract
+  deadline?: moment.Moment,                     // also used to hide the ad after this date
 }
 
 // job-specific data for the ad
 // (also used in the reminder email, so links in the description need to be absolute)
 export const JOB_AD_DATA: Record<string, JobAdData> = {
-  'gwwc-researcher': {
+  // 'iaps-ai-policy-fellowship': {
+  //   careerStages: ['earlyCareer'],
+  //   interestedIn: ['AI strategy & policy'],
+  //   logo: 'https://80000hours.org/wp-content/uploads/2023/10/institute_for_ai_policy_and_strategy_iaps_logo-160x160.jpeg',
+  //   occupation: 'AI policy',
+  //   feedbackLinkPrefill: 'AI+Policy+Fellow+at+IAPS',
+  //   bitlyLink: "https://efctv.org/49Jfdx5", // https://www.iaps.ai/fellowship
+  //   role: 'AI Policy Fellow',
+  //   insertThe: true,
+  //   org: 'Institute for AI Policy & Strategy',
+  //   orgLink: '/topics/institute-for-ai-policy-and-strategy',
+  //   salary: '$5k per month',
+  //   location: 'Remote',
+  //   deadline: moment('2024-03-18'),
+  // },
```

Comment:
I'm still waiting to hear back for this job so I'm commenting it out for now. I will either uncomment or delete this in a later PR.